pkg/controller/status/status.go

```diff
@@ -407,6 +416,15 @@ func (m *statusManager) syncState() bool {
 	return m.explicitDegradedReason != ""
 }
 
+// Remove status displayed for CR
```

Comment:
I think this message could be a little more explicit, mentioning specifically that the "tigera-status" for the CR is removed. You should also take a look at this https://blog.golang.org/godoc-documenting-go-code. The standard for go documentation is to start the comment with the name of the element being described.